.github/workflows/amd-aie-tests.yml

```diff
@@ -29,5 +29,5 @@ jobs:
       build_target: check-llvm-aie check-clang-aie check-lld-aie
       projects: clang;lld
       cache-key: amd-aie
-      extra_cmake_args: '-DLLVM_USE_LINKER=gold -C ../clang/cmake/caches/Peano-AIE.cmake'
+      extra_cmake_args: '-DLLVM_USE_LINKER=gold -C clang/cmake/caches/Peano-AIE.cmake'
```

Comment:
What exactly changed here?

Reply:
Upstream CI made a change to make use of reusable workflow files, and we use those same workflows under the hood.
The cmake config invocation now happens outside of the build directory, i.e. inside the main source repository.
Hence we need to update the relative path to <clang>